src/gg.regression.unity.bots/Runtime/Scripts/StateRecorder/RecordingAndReplayContainerTypes.cs

```diff
@@ -133,6 +135,16 @@ public void WriteToStringBuilder(StringBuilder stringBuilder)
             IntJsonConverter.WriteToStringBuilder(stringBuilder, framesSincePreviousTick);
             stringBuilder.Append(",\"fps\":");
             IntJsonConverter.WriteToStringBuilder(stringBuilder, fps);
+            if (cpuTime.HasValue)
```

Comment:
We should probably just use `WriteToStringBuilderNullable` on `LongJsonConverter` which handles writing `null` into the json payload.  We generally haven't excluded fields from the json when null, instead we write the null.

Reply:
Updated to use this function instead.